test/api-client/api-client.captions.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { RevAiApiClient } from '../../src/api-client';
+import { ApiRequestHandler } from '../../src/api-request-handler';
+import { CaptionTypes } from '../../src/models/async/CaptionTypes';
+import { objectToStream } from '../testhelpers';
+const fs = require('fs');
```

Comment:
Why can't we use the import syntax here?
```suggestion
import * as fs from 'fs';
```